ucte/ucte-converter/src/test/java/com/powsybl/ucte/converter/UcteImporterTest.java

```diff
@@ -39,9 +39,9 @@ public void countryAssociationIssueTest() {
 
         Network network = new UcteImporter().importData(dataSource, null);
 
-        assertEquals(Country.ES, network.getSubstation("EHORTA").getCountry());
+        assertEquals(Country.ES, network.getSubstation("EHORTA").getCountry().orElse(null));
```

Comment:
orElseThow(...) ?

Reply:
Is it really neccassary in tests? The assert will be throw in any case if `getCountry()` returns `null`, won't it?

Reply:
No, I just used to do that. No problem to keep it like this.